src/app/public/modules/infinite-scroll/infinite-scroll-dom-adapter.service.ts

```diff
@@ -115,6 +115,7 @@ export class SkyInfiniteScrollDomAdapterService implements OnDestroy {
     } while (
       !regex.test(style.overflow) &&
       !regex.test(style.overflowY) &&
+      parent.parentNode &&
```

Comment:
☝️ The demo was throwing console errors because this logic runs before the `<sky-docs-demo>` component was added to the DOM, resulting in a null error. This check ensures the `parent` will have a `parentNode` before running that logic.